examples/2d/cifar_small_sample.py

```diff
@@ -225,13 +225,14 @@ def main():
 
     # Optimizer
     lr = 0.1
+    optimizer = torch.optim.SGD(model.parameters(), lr=lr, momentum=0.9,
+                                        weight_decay=0.0005)
     M = args.learning_schedule_multi
     drops = [60*M,120*M,160*M]
     for epoch in range(0, 200*M):
         if epoch in drops or epoch==0:
-            optimizer = torch.optim.SGD(model.parameters(), lr=lr, momentum=0.9,
-                                        weight_decay=0.0005)
-            lr*=0.2
+            for param_group in optimizer.param_groups:
+                param_group['lr'] *= 0.2
```

Comment:
I think you should be consistent and use the same strategy you used previously for updating the LR.

Reply:
I updated the lr differently this time in order to handle the drops, but I will use a MultiStepLR scheduler instead